src/generators/daf_file_generator.py

```diff
@@ -81,15 +81,15 @@ def generate_file(self):
                 evidence_code = disease_association["evidenceCode"]
             else:
                 evidence_code = ""
-            gene_product_form_id = ""
-            additional_genetic_component = ""
-            experimental_conditions = ""
-            qualifier = ""
-            modifier_association_type = ""
-            modifier_qualifier = ""
-            modifier_genetic = ""
-            modifier_experimental_conditions = ""
-            genetic_sex = ""
+            # gene_product_form_id = ""
+            # additional_genetic_component = ""
+            # experimental_conditions = ""
+            # qualifier = ""
+            # modifier_association_type = ""
+            # modifier_qualifier = ""
+            # modifier_genetic = ""
+            # modifier_experimental_conditions = ""
+            # genetic_sex = ""
```

Comment:
I am not sure if these should be deleted. We need to get the latest spec to see if they should be added or not. I am leaning towards deleting them.

Reply:
I just commented out as they are not used ATM, but might be useful in the future.